ui/local_dev.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+MINISHIFT_IP=`minishift ip`
+USER_TOKEN=`oc whoami -t`
+
+if [ -z "$MINISHIFT_IP"]; then
```

Comment:
This does not work on my machine. When minishift is not running, the content of `MINISHIFT_IP` is `Running this command requires a running 'minishift' VM, but no VM is running.`

I changed it to `minishift status | head -1 | awk -F: '/ /{gsub(/ /,  "", $2); print $2}'`: this returns `Stopped` if minishift is stopped, `Running` otherwise